### PR TITLE
feat(v3): init

### PR DESCRIPTION
**Pull request by DjDeveloperr**

### CODE REVIEW

File: .github/workflows/fmt.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Format
+
+on:
+  push:
+    branches: [main, v3]
```
Comment (Helloyunho) [inline .github/workflows/fmt.yml:5]: running ci when `main` is updated?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.vscode/
+.idea/
```
Comment (Helloyunho) [inline .gitignore:2]: we should add `.DS_Store` too, maybe I can do it later

**Review by Helloyunho — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (DjDeveloperr) [reply .gitignore:2]: Added

File: .github/workflows/fmt.yml (hunk 1 above)
Comment (DjDeveloperr) [reply .github/workflows/fmt.yml]: Removed

**Review by Helloyunho — APPROVED** [review]
lgtm!

**Pull request merged**